packages/node-opcua-server/source/server_end_point.ts

```diff
@@ -245,6 +305,9 @@ export class OPCUAServerEndPoint extends EventEmitter implements ServerSecureCha
 
         this.port = parseInt(options.port.toString(), 10);
         assert(typeof this.port === "number");
+        if (options.host !== undefined) {
+            this.initializeHost(options.host);
```

Comment:
As we are using a separated `host` parameter different from the `hostname` there is no need to call `initializeHost` any more, as we can assume that the variable will contain the IP address of hostname that resolve to the network interface that we want to use already.

initializeHost not been used, this will simplify this commit a lot, 

No need for validateHost and isLoopback address anymore.